deepchecks/vision/checks/distribution/train_test_label_drift.py

```diff
@@ -10,23 +10,58 @@
 #
 """Module contains Train Test label Drift check."""
 from copy import copy
-from typing import Dict, Hashable, Callable, Tuple, List, Union
+from typing import Dict, Hashable, Callable, Tuple, List, Union, Any
 
 from plotly.subplots import make_subplots
 
 from deepchecks import CheckResult, ConditionResult
-from deepchecks.vision.base import TrainTestCheck, Context
+from deepchecks.vision.base import Context, TrainTestCheck
 from deepchecks.utils.distribution.plot import drift_score_bar_traces
 from deepchecks.utils.plot import colors
 from deepchecks.vision.dataset import VisionDataset, TaskType
 import numpy as np
 from collections import Counter
 import plotly.graph_objs as go
 
-
 __all__ = ['TrainTestLabelDrift']
 
 
+# TODO: Add label sampling when available
+
+# Functions temporarily here, will be changed when Label and Prediction classes exist:
+def get_bbox_area(label):
```

Comment:
@ItayGabbay These probably should be moved to your new code for format transformations.